src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Ccda/CCD/CCD-expected.json

```diff
@@ -583,19 +583,6 @@
             }
           ]
         },
-        "reaction": [
```

Comment:
Are we removing this property?

Reply:
according to https://www.hl7.org/fhir/allergyintolerance.html, there should have one manifestation in reaction, if there is not manifestation(entryRelationShip), I think we should also don't have "substance"